crates/orchestrator/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "orchestrator"
+version = "0.0.1"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[dependencies]
+glob = "0.3.1"
+color-eyre = "0.6.2"
+russh = "0.38.0"
+russh-keys = "0.38.0"
+aws-sdk-ec2 = "0.29.0"
+aws-smithy-http = "0.56"
+aws-smithy-runtime-api = "0.56"
```

Comment:
Can we place these in the root toml so that these deps are easier to maintain and update? (same with the prometheus-parse dep below)